docs/website/docs/dlt-ecosystem/file-formats/parquet.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Parquet
+description: The parquet file format
+keywords: [parquet, file-formats]
+---
+
+# Parquet file format
+
+[Apache Parquet](https://en.wikipedia.org/wiki/Apache_Parquet) is a free and open-source column-oriented data storage format in the Apache Hadoop ecosystem. dlthub is able to store data in this format when configured to do so.
+
+To use this format you need a `pyarrow` package. You can get this package as an `dlt` extra as well:
+```sh
+pip install dlt[pyarrow]
+```
+
+## Supported destinations
+Used by default by: **filesystem**
+
+Supported by: **BigQuery**, **DuckdDb**, **Snowflake**
+
+By setting the `loader_file_format` argument to `parquet` in the run command, the pipeline will store your data in the parquet format to the destination:
+
+```python
+info = pipeline.run(some_source(), loader_file_format="parquet")`
+```
+
+
+## Options
+
+Under the hood dlt uses the [pyarrow parquet writer](https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetWriter.html) to create the files. The following options can be used to change the behavior of the writer:
+
+* `flavor`: Sanitize schema or set other compatibility options to work with various target systems. Defaults to spark.
```

Comment:
maybe it would be appropriate to add what other options there might be?

Reply:
not documented in their docs either

Reply:
@AstrakhantsevaAA this is all there is. there's more on performance that I improved as well. do you want to link to general options?

Reply:
okay, I'll link to performance page